### ISSUE TITLE
Turn out too well

### ISSUE THREAD

**Issue by TiAmo888** [aikunyi/FreTS#10]
Why are your results so good? Even if the method you proposed is removed, the prediction results are still very good using only the full connection layer. I suspect that there may be problems with the method you proposed

**Comment by jackyue1994** [comment]
Yeah, it seems that FreMLP is not functioning as intended when it operates on the embed_size dimension. It fails to learn any inter-series or intra-series dependencies. See [ issue 12. ](https://github.com/aikunyi/FreTS/issues/12)

**Comment by vincent05r** [comment]
did you keep the dimension extension layer? if so that might be very interesting since it suggests that the linear layer in it does some very heavy lifting.
